infrastructure/variables.tf

```diff
@@ -25,11 +25,11 @@ variable "s2s-url" {
 }
 
 variable "ccd-url" {
-  default = "https://case-data-app.test.ccd.reform.hmcts.net:4481"
+  default = "http://ccd-data-store-api-sandbox.service.core-compute-sandbox.internal"
```

Comment:
move this to main.tf, look at the existing samples like the pdf service

Reply:
Are you sure? This is where you had it originally? Why do you want me to change this away from where you had it?